src/com/ableton/Pipenv.groovy

```diff
@@ -1,54 +0,0 @@
-package com.ableton
```

Comment:
Please reword the commit message of caf0c2d05158d126ee013e34e751b65ee8e7c9da to remove all subjective views on pipenv. We don't use it anymore, but that doesn't mean anything about the project itself.

> current pipenv releases (if there are any).

There are many: https://github.com/pypa/pipenv/releases

Reply:
See 49b0b28.

Reply:
Thanks